crates/wasmparser/src/validator/types.rs

```diff
@@ -277,9 +287,11 @@ impl Type {
     }
 
     pub(crate) fn type_size(&self) -> u32 {
+        // TODO calculate actual size for func, array, struct (#1036)
```

Comment:
Can you file a follow up issue to track this?

```suggestion
        // TODO(#1234): calculate actual size for func, array, struct (#1036)
```

Reply:
Is there anything that #1036 isn't covering?